msb/rawlogger/msb_rawlogger.py

```diff
@@ -32,20 +33,25 @@ def get_data(self) -> tuple[bytes, bytes]:
                 continue
             yield topic, data
 
-    def run(self):
-        for topic, data_raw in self.get_data():
-            if self.config.print_stdout:
-                print(f"{topic.decode('utf-8')} : {data_raw.decode('utf-8')}")
 
-            data = b'{"' + topic + b'":' + data_raw + b"}\n"
-            self.logger.write(data)
+    def run(self,excluded_topics):
+        for topic, data_raw in self.get_data(excluded_topics):
+            if topic.decode('utf-8') in excluded_topics:
```

Comment:
Decode the topic only once and save it in a variable.

Reply:
Consider using `if topic not  in excluded_topics:` instead. Then you will not need the `pass` and else statements.

Reply:
Incorperated your suggestion.